crates/pallet-feeds/src/lib.rs

```diff
@@ -52,17 +63,28 @@ pub mod pallet {
         ) -> DispatchResult {
             let who = ensure_signed(origin)?;
 
+            let object_size = data.len() as u64;
+
             // TODO: add data handling
```

Comment:
What is this TODO about?

Reply:
it's an old one - currently, we don't do anything with the actual data, can remove if you think it's redundant